GitUI/Themes/darksilver.css

```diff
@@ -38,13 +38,14 @@
 .HighlightAllOccurences  { color: #606020; }
 .Tag                     { color: #408080; }
 .Graph                   { color: #882d00; }
-.RemoteBranch            { color: #ff8080; } /* hsl(0,   100%, 75%) */
+.Branch                  { color: #00d76b; }
 .DiffSection             { color: #4f4f4f; }
 .DiffAdded               { color: #054e2a; }
 .DiffAddedExtra          { color: #058545; }
 
-.Branch                      { color: #00d76b; }
-.Branch.colorblind           { color: #0080ff; } /* hsl(210, 100%, 50%) */
+
+.RemoteBranch                { color: #ff8080; } /* hsl(0,   100%, 75%) */
```

Comment:
nit pick and not from this PR: Can the colors be ordered as in the other themes so they are easier to compare?

Reply:
done